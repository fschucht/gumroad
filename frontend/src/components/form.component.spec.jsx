import renderer from 'react-test-renderer';
import { Form } from './form.component';

describe('Form', () => {
  it('should render a Form component', () => {
    const renderedComponent = renderer.create(<Form>Hello World!</Form>).toJSON()

    expect(renderedComponent).toMatchSnapshot()
  })
})