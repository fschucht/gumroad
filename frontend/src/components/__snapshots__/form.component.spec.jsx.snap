// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Form should render a Form component 1`] = `
<form
  className="form"
>
  Hello World!
</form>
`;
